src/Arcus.Messaging.Health/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -16,16 +16,28 @@ public static class IServiceCollectionExtensions
         ///     it here or do it yourself as well.
         /// </remarks>
         /// <param name="services">Collection of services to use in the application</param>
+        /// <param name="tcpConfigurationKey">The TCP health port on which the health report is exposed.</param>
```

Comment:
Nitpick - Maybe be a bit more explicit?
```suggestion
        /// <param name="tcpConfigurationKey">Configuration key that defined the TCP health port on which the health report is exposed.</param>
```

Reply:
Yep, done.